### ISSUE TITLE
Featured links could provide more specific support for claims

### ISSUE THREAD

**Issue by mwaskom** [matplotlib/mpl-brochure-site#10]
First of all, this page looks fantastic. I'm really digging the embrace of viridis as a visual identity for matplotlib!

As mentioned on gitter, I have some feedback about the choice of targets for some of the featured links. (Apologies if this is obvious and you are just thinking of these as placeholders while you iterate on the design!)

Specifically I'm thinking about this section:

![image](https://user-images.githubusercontent.com/315810/136676780-45e1342f-8de6-44bf-a1a8-abe5221c9d6c.png)

This is prime real estate and these links will get thousands of clicks a day, many from people who are totally new to the project and want to understand what it's about. I think the bullet points advertise the key selling points of matplotlib very well. But the links in those bullets take users to parts of the docs that don't support the claims in a particularly specific or useful manner.

Going through them:

"Create publication quality plots" leads to the main example gallery. Opinions differ about what "publication quality" means, but many of the plots you immediately see are very basic demos, lacking features that you'd need to have in a publication (e.g. axis labels), and none demonstrate the polish that matplotlib is able to give to figures with a little effort. I've published plenty of figures that are 100% end-to-end matplotlib and this is absolutely a selling point, but poorly demonstrated by this link imo.

(It's also a bit overwhelming: you're on this lovely landing page, and then the first link takes you to an inventory of examples without a whole lot of narrative / organizational structure.)

"Make interactive figures that can zoom, pan, update." Also a selling point! But linking to the event handling examples makes this maybe seem more complicated than it actually is. With a GUI backend your plots zoom, pan, and update for free — matplotlib *also* lets you define custom interactions, which is what these examples demonstrate.

"Customize visual style and layout." This links to the homepage for the tutorials. I suppose that's a pretty general claim and so the tutorials are as good a place to go as any, but it's not especially demonstrative and I wonder about whether you want a user getting lost in the capacious docs on before reading more than a couple of bullets of the homepage.

"Export to many file formats" and "Embed in JupyterLab and Graphical User Interfaces" both link to the backend API docs, which are relevant but not in a way that's helpful or illustrative to someone new to the project.

"Use domain-specific plots from third-party packages." Perfect. No notes :)

---

IMO I am not sure that you need or want links on these bullet points. A lot of users are going to click on them and find themselves in the main docs without the helpful signposting that's on the lower half of the page, and I think you want a new user to get a high-level summary of what matplotlib offers before diving in.

But if you feel strongly that these claims need justification / demonstration, it would probably be worth thinking about what the best specific page to link to for each of them is. (Perhaps the lack of a specific target for some of these links is also a signal about what could be added to the docs. For instance, *do* the docs have a nice self-contained tutorial on "making (polished) publication-quality figures"?)

---

But again: page looks great, and I think people will really benefit from the work y'all are putting into this.

**Comment by story645** [comment]
Do you think there are sections in the examples gallary or tutorials that would be better to link to? 

Alternatively, should there be new docs/tutorials, something like?
* Out of the box interactivity - pan, brush, zoom (basically what is sharex, how to set the data cursor, what are the widgets - the most basic out of the box built into mpl/the toolbar stuff)
* Publication quality plots - though what exactly would go here (for example if the gallery could be curated via tags?)

**Comment by mwaskom** [comment]
> However, I think the main goal is to make people aware that there are tailored solutions for specific categories of plots and they do not necessarily have to tailor their specific plots using core matplotlib.

Yeah I don't think it's a big problem one way or the other it just reads as slightly weird to me. Matplotlib doesn't make it possible for you use third party tools exactly ... it makes it possible for those tools to exist, and then you use them.

**Comment by timhoffm** [comment]
Since 99.999% of visitors will be users, not authors of 3rd party libraries, I believe this most prominently placed bullet list should address their interests and not confuse them by to explanations they do not need.

Maybe we can refine the wording.

> Use domain-specific plots from third-party packages built on top of Matplotlib.

or

> Use domain-specific plots from packages extending Matplotlib.

**Comment by timhoffm** [comment]
> Matplotlib is the foundation for a rich ecosystem of visualization packages

Is ok. I'm not over-excited because in contrast to the other bullet points it just states a fact and does not tell the user how the fact relates to him. But when including that we either are at @story645's suggestion

> Use domain specific plot from packages built on top of Matplotlib
> Build your own reusable domain specific libraries and applications

which I find a bit long (and reiterating from above, when setting priorities, advertising to everybody to actively build libraries on top of Matplotlib is not a title-page goal to me). 

> FWIW I think a user who sees a bullet advertising that matplotlib can be extended by domain specific interfaces, along with a showcase of libraries that do that, will probably get the right takeaway, even if the bullet is technically addressed to potential authors of those libraries.

I'd like to invert the statement: The few new (potential) library authors are reasonably experienced developers. They will realize that they can write a library on top of Matplotlib if they see that there are a number of those already.


#### In summary:
I'd be happy with either

> Matplotlib is the foundation for a rich ecosystem of visualization packages

or 

> Use domain specific plots from packages built on top of Matplotlib

**Comment by tacaswell** [comment]
I think this was closed by #21
